easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-6.1.0-CrayIntel-17.08-pat-6.5.1.eb

```diff
@@ -23,13 +23,14 @@ builddependencies = [
     ('perftools-lite/%s' % patversion, EXTERNAL_MODULE)
 ]
 
-preconfigopts = ' CC=cc LDFLAGS+="$LDFLAGS -qopenmp" && ' 
+preconfigopts = ' unset CRAYPAT_COMPILER_OPTIONS && '
```

Comment:
You could also try with:

```
preconfigopts = 'unset CRAYPAT_LITE &&'                                         
buildopts = ' CRAYPAT_LITE=sample_profile '               
```

Reply:
If I set `CRAYPAT_LITE=sample_profile` when building, can I change profile later at runtime?